wallet/commands.go

```diff
@@ -73,76 +16,112 @@ func (wal *Wallet) HaveAddress(address string) (bool, string) {
+// 	if txnBlockHeight != -1 && confirmations > dcrlibwallet.DefaultRequiredConfirmations {
+// 		return "confirmed", confirmations
+// 	}
+// 	return "pending", confirmations
+// }
+
+// // WalletSyncStatus returns the sync status of a single wallet
+// func walletSyncStatus(isWaiting bool, walletBestBlock, bestBlockHeight int32) string {
+// 	if isWaiting {
+// 		return "waiting for other wallets"
+// 	}
+// 	if walletBestBlock < bestBlockHeight {
+// 		return "syncing..."
+// 	}
+
+// 	return "synced"
+// }
+
+// func (wal *Wallet) UnlockWallet(walletID int, password []byte) error {
+// 	return wal.multi.UnlockWallet(walletID, password)
+// }
+
+// // CurrentAddress returns the next address for the specified wallet account.
+// func (wal *Wallet) CurrentAddress(walletID int, accountID int32) (string, error) {
+// 	wall := wal.multi.WalletWithID(walletID)
+// 	if wall == nil {
+// 		return "", ErrIDNotExist
+// 	}
+// 	return wall.CurrentAddress(accountID)
+// }
 
-func (wal *Wallet) ReadBoolConfigValueForKey(key string) bool {
-	return wal.multi.ReadBoolConfigValueForKey(key, false)
-}
+// NextAddress returns the next address for the specified wallet account.
+// func (wal *Wallet) NextAddress(walletID int, accountID int32) (string, error) {
+// 	wall := wal.multi.WalletWithID(walletID)
+// 	if wall == nil {
+// 		return "", ErrIDNotExist
+// 	}
+// 	return wall.NextAddress(accountID)
+// }
+
+// // IsAddressValid checks if the given address is valid for the multiwallet network
+// func (wal *Wallet) IsAddressValid(address string) (bool, error) {
+// 	return wal.multi.IsAddressValid(address), nil
+// }
 
-func (wal *Wallet) ReadStringConfigValueForKey(key string) string {
-	return wal.multi.ReadStringConfigValueForKey(key)
-}
+// VerifyMessage checks if the given message matches the signature for the address.
+// func (wal *Wallet) VerifyMessage(address string, message string, signature string) (bool, error) {
+// 	return wal.multi.VerifyMessage(address, message, signature)
+// }
 
-func calculateDaysBehind(lastHeaderTime int64) string {
-	diff := time.Since(time.Unix(lastHeaderTime, 0))
-	daysBehind := int(math.Round(diff.Hours() / 24))
-	if daysBehind < 1 {
-		return "<1 day"
-	} else if daysBehind == 1 {
-		return "1 day"
-	} else {
-		return fmt.Sprintf("%d days", daysBehind)
-	}
-}
+// StartSync starts the multiwallet SPV sync
+// func (wal *Wallet) StartSync() error {
+// 	return wal.multi.SpvSync()
+// }
 
-// SecondsToDays takes time in seconds and returns its string equivalent in the format ddhhmm.
-func SecondsToDays(totalTimeLeft int64) string {
-	q, r := divMod(totalTimeLeft, 24*60*60)
-	timeLeft := time.Duration(r) * time.Second
-	if q > 0 {
-		return fmt.Sprintf("%dd%s", q, timeLeft.String())
-	}
-	return timeLeft.String()
-}
+// RescanBlocks rescans the multiwallet
+// func (wal *Wallet) RescanBlocks(walletID int) error {
+// 	return wal.multi.RescanBlocks(walletID)
+// }
```

Comment:
Would remove the entire file in subsequent code clean up… I feel the file is has served it’s use but it won’t be in this PR. Left the comments for reference just in case.